app/models/user.rb

```diff
@@ -432,6 +432,8 @@ def send_welcome_message
     return unless AppConfig.settings.welcome_message.enabled? && AppConfig.admins.account?
     sender_username = AppConfig.admins.account.get
     sender = User.find_by(username: sender_username)
+    # Check if admin account is no longer valid user per Issue #7363
```

Comment:
I don't think we need a comment for that, the check should be pretty self-explaining.